src/containers/ResourceForm/ResourceForm.jsx

```diff
@@ -30,6 +30,7 @@ class ResourceForm extends Component {
     this.state = {
       uiResource: props.uiResource,
       formData: props.formData,
+      formId: Date.now(),
```

Comment:
Is not easy to understand why an id is set on an timestamp. Maybe something like 'id' + Date.now()?